app/views/cdo/create/address.njk

```diff
@@ -48,6 +49,8 @@
         
           {{ govukInput(model.postcode) }}
 
+          {{ govukSelect(model.countries) }}
```

Comment:
Being picky this name should be 'model.country' as it's a single field (select field)